### ISSUE TITLE
name correction suggestion for [white-eye]

### ISSUE THREAD

**Issue by RuiyaLI** [globalbioticinteractions/globi-taxon-names#5]
Hi!
I found a taxon name ```white-eye``` on a [globi page](https://www.globalbioticinteractions.org/?interactionType=interactsWith&sourceTaxon=NCBI%3A1159907). After reviewing the data sources and relevant taxonomic tools like the [globalnames resolver](https://resolver.globalnames.org), I think that suggestion below might be helpful.

 Thanks!

| name | suggested name | name url reference | notes
| --- | --- | --- | ---
| white-eye | YOUR SUGGESTED NAME | http://somereference | some notes

**Comment by jhpoelen** [comment]
hi @RuiyaLI - thanks for pointing out this suspicious taxonomic name "white-eye" . 

After tracing the provenance / source of the interaction claim, I found that an incomplete transcription to be the likely source of the confusing name.

In https://zenodo.org/record/3949076 , Plazi included an annotation (see attached screenshot):

 ( white-eye )- [:host of]→ ( white-eye coronavirus HKU16, WECoV HKU16 ) . 

![Screenshot from 2020-11-13 08-26-14](https://user-images.githubusercontent.com/1084872/99096783-8fe89b80-258b-11eb-860f-46e2b69b1857.png)


extracted from:

Woo, Patrick C. Y., Lau, Susanna K. P., Lam, Carol S. F., Lau, Candy C. Y., Tsang, Alan K. L., Lau, John H. N., … Yuen, Kwok-Yung. (2012). Discovery of seven novel Mammalian and avian coronaviruses in the genus deltacoronavirus supports bat coronaviruses as the gene source of alphacoronavirus and betacoronavirus and avian coronaviruses as the gene source of gammacoronavirus and deltacoronavirus. Journal of Virology, 86(7), 3995–4008. http://doi.org/10.1128/JVI.06540-11

On close inspection of the source, I believe that the annotation should have been:

 ( Japanese white-eye )- [:host of]→ ( white-eye coronavirus HKU16, WECoV HKU16 ) . 

where a Japanese white-eye appears to be a common name for Zosterops japonicus , a bird.

@myrmoteras @mguidoti please let me know if you have anyway to have a look at this suspicious transcribed name in https://zenodo.org/record/3949076 .

**Comment by mguidoti** [comment]
Yes, but before I fix it, I'll check with Zenodo team what's the best approach to this as they do not list custom metadata on the edit page on their interface... 

But we're on it.

**Comment by jhpoelen** [comment]
@mguidoti excellent! Am curious to hear your ideas on making this feedback loop as pragmatic as we can.
